src/tests/cli/firewall-cmd.at

```diff
@@ -1175,6 +1175,12 @@ FWD_START_TEST([direct])
 FWD_END_TEST([-e '/ERROR: INVALID_TABLE:/d' dnl
               -e '/WARNING: NOT_ENABLED: chain/d'])
 
+FWD_START_TEST([direct zone])
```

Comment:
This new test is in response to a bug report. I'd rather see it added under `src/tests/regression`.